content/english/anti-vegan/humans-are-superior.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Humans Are Superior"
+date: 2019-10-12T07:28:15+13:00
+name: tery-hardwicke
+contributors:
+    - 
+        name: tery-hardwicke
+        role: author
+    - 
+        name: sam-martin
+        role: editor
+draft: false
+description: "How to respond to the argument: We are entitled to use animals as a resource because humans are superior to animals"
+weight: 5
+---
+# Argument
+
+> ...but, humans are superior...  
+>   
+> We are entitled to use animals as a resource because humans are superior to animals
+
+# Rebuttal
+
+## Quick Responses
+
+> Superiority supposes an arbitrary line that identifies one group as deservedly superior to another. Common examples are that men are superior to women (sexism) and one ethnicity is superior to all other ethnicities (racism).  
```

Comment:
I think this lacks something as an independent response as the next step of the rebuttal needs spelling out. As I guess the point is "Do you support those forms of bigotry? Do you have a reason that your assertion is meaningfully different from those forms of bigotry?"

Also shall we just say "Common examples are, sexism, racism, etc. all bigotry essentially" as I'd like to minimize the number of out-of-context snippets in the text that look like:

> men are superior to women

or

> one ethnicity is superior to all other ethnicities

Reply:
Actually don't think the examples are needed here at all.

Reply:
Then I'll just replace first or with an 'and'

Reply:
Logical falsehood first the the three sub-parts